gamemaker.py

```diff
@@ -139,19 +141,37 @@ def set_penalties(self, input):
             ret_string = "Bad spacing! Need one space between time penalty and guess penalty"
```

Comment:
Should also remind the user here that time and guesses must be integers in the except block, in case they're passing in some other type for whatever reason. Exceptions will be thrown in this case as well.

Reply:
If they try to input an invalid data type, that's already handled. Not sure what else you're getting at...